### ISSUE TITLE
GIF compression

### ISSUE THREAD

**Issue by Sarisan** [tdlib/telegram-bot-api#646]
`InlineQueryResultGif` and `sendDocument` both compress GIF passed with URL. Can this annoying behavior be disabled any way?

**Comment by levlam** [comment]
No. All animations are intended to be played in MP4 format, which can provide the same or better quality with much smaller file size.

**Comment by Sarisan** [comment]
> No. All animations are intended to be played in MP4 format, which can provide the same or better quality with much smaller file size.

Provide the compression option then where user can pick the quality like in android client. Or turn this compression off since either of clients degrades quality during GIF to MP4 conversion.
